### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Uzair-Manzoor**
## This pull request has been created against implementation of the below mentioned requirements into the project:

- [x] Add any **missing JSX elements** to your project (without adding extra functionality)
- [x] Your final design **should match Zeplin's design** (this includes all elements + the same styling)
- [x] You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
+      <nav className="navbar">
+        <h2 className="logo">Bookstore CMS</h2>
+        <ul>
+          <li className="books-link">
+            <NavLink to="/">BOOKS</NavLink>
+          </li>
+          <li className="categories">
+            <NavLink to="/categories">CATEGORIES</NavLink>
+          </li>
+        </ul>
```
Comment (jerryowusu) [inline src/components/header.js:16]: Your project is almost complete. Good job so far. However, kindly make sure that your design matches the actual one in terms of color. The color of your design looks slightly different to the actual design as shown below. Kindly ensure you use the specified color of #121212

### Your design 

Screenshot from 2023-09-29 19-38-17

### Actual design 

Screenshot from 2023-09-29 19-34-37

File: src/components/form.js [hunk 2]
```diff
@@ -36,17 +36,21 @@ const Form = () => {
...
-        ))}
-      </select>
-      <button type="submit">ADD BOOK</button>
-    </form>
+    <div className="form-section">
+      <hr />
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <form onSubmit={handleFormSubmit}>
+        <input type="text" placeholder="Book title" value={title} onChange={handleTitleChange} required />
+        <input type="text" placeholder="Author" value={author} onChange={handleAuthorChange} required />
```
Comment (jerryowusu) [inline src/components/form.js:44]: The placeholder of the `book title` and the `author` looks slightly different to the actual design as shown below. Kindly ensure you use the specified color of #c4c4c4 and nothing else to get the actual design specified. 

### Your design 

Screenshot from 2023-09-29 19-38-24


### Actual design 
Screenshot from 2023-09-29 19-51-22

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Uzair-Manzoor  

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

so-close


## Highlights
Nice design :+1: 
Api working fine :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @Uzair-Manzoor, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Highlights
- Great job, your project is error free :+1: 
- Nice work you meet all the requirements :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**